g/GitHub Actions.yaml

```diff
@@ -0,0 +1,11 @@
+name: Hello World
+
```

Comment:
There should only be a single newline here

Reply:
Done, thanks!